Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -118,6 +119,14 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication(EdFiAuthenticationTypes.OAuth)
                 .AddScheme<AuthenticationSchemeOptions, EdFiOAuthAuthenticationHandler>(EdFiAuthenticationTypes.OAuth, null);
+ 
+            services.AddAuthorization(options =>
```

Comment:
The way you had it before was correct, you should only add this policy if Identity is enabled.